### PR TITLE
Update iotex API, add iotex test address to default list[iotex-staked-balance]

### PR DESCRIPTION
**Pull request by millken**
Fixes # .

Changes proposed in this pull request:
- Update iotex API endpoints and func name.

### CODE REVIEW

**Comment by millken** [comment]
@ChaituVR

**Comment by ChaituVR** [comment]
@maswalker Can you confirm these changes?

**Comment by guo** [comment]
Hello, @ChaituVR !

This is Guo from IoTeX. We have run the test locally and it worked. We suspect that the endpoint might be slow during the last run. Could you please re-run the test here?  

Thank you!

**Comment by bonustrack** [comment]
@millken Tests are failing, can you have a look on the error and try to fix it? Check here https://github.com/snapshot-labs/snapshot-strategies/actions/runs/3047847939/jobs/4913284908

**Review by maswalker — APPROVED** [review]

**Comment by guo** [comment]
@bonustrack @ChaituVR  could you please rerun the test? We have improved the endpoint. It is passing when we test locally.

**Comment by guo** [comment]
The tests are passed now. @ChaituVR, could you please review and merge the PR? Thank you!

**Comment by ChaituVR** [comment]
Hey @guo `iotex-balance` strategy works fine now. but not `iotex-staked-balance` 

```js
Test strategy "iotex-staked-balance" with example index 0 (latest snapshot)
    ✓ Strategy should run without any errors (946 ms)
    ✕ Should return an array of objects with addresses (3 ms)
```

I understand it is urgent. but it should work with the latest snapshot too. maybe the addresses passed into examples.json doesn't have VP for the latest block? can fix it quickly?

File: src/strategies/iotex-balance/index.ts [hunk 1]
```diff
@@ -5,10 +5,10 @@ interface ApiReturn {
 }
 
 export const author = 'iotex';
```
Comment (ChaituVR) [inline src/strategies/iotex-balance/index.ts:7]:
```suggestion
export const author = 'iotexproject';
```

File: src/strategies/iotex-staked-balance/index.ts [hunk 2]
```diff
@@ -4,10 +4,10 @@ interface ApiReturn {
 }
 
 export const author = 'iotex';
```
Comment (ChaituVR) [inline src/strategies/iotex-staked-balance/index.ts:6]:
```suggestion
export const author = 'iotexproject';
```

**Comment by millken** [comment]
@ChaituVR It has already been fixed, all test should be passed.

**Review by ChaituVR — APPROVED** [review]

**Comment by guo** [comment]
@ChaituVR , thanks for the quick review!

**Pull request merged**